tests/testthat/test-utils.R

```diff
@@ -72,3 +72,15 @@ test_that("rename_118() function is not applied if issue #118 is not addressed",
   expected_col_name <- "PSTR_risk_category"
   expect_equal(grep(actual_col_name, names(result), value = TRUE), expected_col_name)
 })
+
+test_that("`stop_if_percent_noise_more_than_100` throws an error if added noise is more than 100%", {
+  data <- tibble(
+    min = 1,
+    max = 2,
+    max_jitter = 10000,
+    min_jitter = 10000
+  )
+
+  expect_error(stop_if_percent_noise_more_than_100(data), "noise.*readjust.*amount")
+})
```

Comment:
Ideally we would test this behaviour using the public API -- i.e. one of the `profile*()` functions.

Reply:
Honestly, I couldn't find an example which could throw the exact same error we want this function `stop_if_percent_noise_more_than_100` to throw. The `jitter_range` function always create a value below 100% threshold. That's why I choose a random sample to just test its functionality. I know that's not the right way to test this use case on the `profile*` function. I now think that this test is not required. What do you think?

Reply:
It relates to my other comment about `amount`. We won't be able to generate a test that triggers this error until we allow changing the `amount` -- maybe via an option as I suggested below. Then it should be possible to find values of `co2_footprint` and `amount` that are crazy enough to trigger this error. 

When a function is easy to test, it's easy to use -- because a test IS a user of the function.